tools/runner/runner.go

```diff
@@ -59,11 +61,12 @@ func NewRunner(loadTestGetter clientset.LoadTestGetter, afterInterval func(), re
 		afterInterval:         afterInterval,
 		retries:               retries,
 		deleteSuccessfulTests: deleteSuccessfulTests,
+		podLogger:             NewPodLogger(),
 	}
 }
 
 // Run runs a set of LoadTests at a given concurrency level.
-func (r *Runner) Run(ctx context.Context, configs []*grpcv1.LoadTest, suiteReporter *TestSuiteReporter, concurrencyLevel int, done chan<- *TestSuiteReporter) {
+func (r *Runner) Run(ctx context.Context, configs []*grpcv1.LoadTest, suiteReporter *TestSuiteReporter, concurrencyLevel int, podLogDir string, done chan<- *TestSuiteReporter) {
```

Comment:
I think the argument here should be `outputDir` instead of `podLogDir`, to be more generic.

Reply:
Sounds good, I also changed the argument of the same name for runTest